### PR TITLE
Add pinyin hint when typing, with shortcut [Ctrl +Period]

### PR DESCRIPTION
**Pull request by jack2game**

### CODE REVIEW

File: moran.schema.yaml [hunk 1]
```diff
@@ -39,6 +39,8 @@ switches:
     states: [ 🈚, 🈶 ]
   - name: chaifen
     states: [ 不拆, 扌斥 ]
+  - name: pinyinhint
+    states: [ 🔇, 🔊 ]
```
Comment (ksqsf) [inline moran.schema.yaml:43]: 👍

**Review by ksqsf — COMMENTED** [review]
Thank you!

On the data: where is the original source of moran_pinyinint.txt? If it is generated, would you please add downloading/generation to tools/Makefile?

**Comment by jack2game** [comment]
No prob. So this was firstly mentioned in rime-ice [here](https://github.com/jack2game/rime-ice/blob/0d93df2174fbebfac67bbf656becf9eca49c7d3c/cn_dicts/41448.dict.yaml#L11)

Which means that I traced to this file https://github.com/mozillazg/pinyin-data/blob/master/zdic.txt

The file was cleaned up a bit into a openCC-friendly format in my PR, but I didn't change any codes for any characters.

**Comment by jack2game** [comment]
btw I didn't write a script to generate the file, instead I just used Excel, based on the assumption that such data should be relatively static.

**Review by ksqsf — APPROVED** [review]

**Pull request merged**